simpa_examples/perform_image_reconstruction.py

```diff
@@ -18,6 +18,7 @@
 
 settings.set_reconstruction_settings({
     Tags.RECONSTRUCTION_PERFORM_BANDPASS_FILTERING: False,
+    Tags.BANDPASS_FILTER_METHOD: Tags.TUKEY_BANDPASS_FILTER,
```

Comment:
Would it make sense to assume a default bandpass filter, if the user requests PERFORM_BANDPASS_FILTERING: True?
Otherwise, it might not be intuitive why bandpass filtering is not performed even though it is requested?

Reply:
I agree, the current solution is not optimal. I think there are two options:

1.  Either have a PERFORM_BANDPASS_FILTERING tag and if it is set to True, then a default filtering method will be applied unless the user specifies another one.
2. We don't have a PERFORM_BANDPASS_FILTERING tag and instead just a BANDPASS_FILTERING_METHOD tag. If a method is set, then this will be used. If none is set, no filtering is performed.

For me, both solutions are fine. But perhaps the first one is better for backwards compatibility.

Reply:
I think I would prefer option 1 because first-time users might not know which filtering methods there are which leaves more room for errors